src/CredentialsImport.js

```diff
@@ -58,6 +58,11 @@ class CredentialsImport {
         this.device.deviceApi.notify(new CloseAutofillParentCall(null))
         this.device.deviceApi.notify(new StartCredentialsImportFlowCall({}))
     }
+
+    async dismissed () {
+        this.device.deviceApi.notify(new CloseAutofillParentCall(null))
+        this.device.deviceApi.notify(new CredentialsImportFlowPermanentlyDismissedCall(null))
```

Comment:
Just a thought. Here we call close, then the other call (same as we do above in line 58). Is there a race condition where the first call's side effect is so quick that the second call never happens because then the webview gets closed and the js context is destroyed? Should we flip their order?

Reply:
That's a good point! I think it's safer to do flipped order. Changed it in 79c1941d4dcd339e49b0bd1a5dd723b31772a496